.github/workflows/validations.yml

```diff
@@ -39,29 +39,13 @@ jobs:
       && github.event.issue.pull_request != null
       && startsWith(github.event.comment.body, '/run_checks')
     steps:
-      - uses: actions/github-script@v2
+      - uses: sushichop/action-repository-permission@v1
         with:
-          script: |
-            github.reactions.createForIssueComment({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              comment_id: context.payload.comment.id,
-              content: "+1"
-            })
-      - uses: actions/github-script@v2
-        id: pr-details
-        with:
-          script: |
-            const { data: pullRequest } = await github.pulls.get({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              pull_number: context.issue.number
-            });
-            return pullRequest;
+          required-permission: write
+          reaction-permitted: '+1'
       - uses: actions/checkout@v2
         with:
-          repository: ${{fromJSON(steps.pr-details.outputs.result).head.repo.full_name}}
-          ref: ${{fromJSON(steps.pr-details.outputs.result).head.ref}}
+          ref: refs/pull/${{ github.event.issue.number }}/head
```

Comment:
I'm not sure that this will work in all situations. I've manually fetched the PR to be independent of any forking situation (e.g. what about 2+ level forks coming back with a PR into upstream).

Reply:
I see. I certainly hadn't thought of that case... So I'd like to revert these line(check out the PR form the fork)